transform.Rmd

```diff
@@ -545,14 +545,21 @@ A better ranking would also consider cancellations, and need a way to convert th
 
 
 `r BeginQuestion()`
-Come up with another approach that will give you the same output as `not_canceled %>% count(dest)` and `not_canceled %>% count(tailnum, wt = distance)` (without using `count()`).
+Come up with another approach that will give you the same output as `not_cancelled %>% count(dest)` and `not_cancelled %>% count(tailnum, wt = distance)` (without using `count()`).
 `r EndQuestion()`
 
 
 `r BeginAnswer()`
 
+Count will group a dataset on the given variable and then determine the number of instances within each group. The easiest way to do this is to `group_by` the given variable and then find the length of each group.  To make it match count, the value should go in a new column `n`.
+```{r}
+not_cancelled %>% group_by(dest) %>% summarize(n = length(dest))
+```
 
-
+For a weighted count, just take the sum of the weight variable in each group.
```

Comment:
Remove "just". Not a good word to use in solutions or instructional materials. See <https://carpentries.github.io/instructor-training/03-expertise/>.